src/main/java/org/isf/utils/db/Auditable.java

```diff
@@ -56,8 +57,9 @@
     @Column(name="LAST_MODIFIED_DATE")
     protected LocalDateTime lastModifiedDate;
    
+	@NotNull
     @Column(name="ACTIVE")
-    protected Integer active = 1;
+    protected Integer active = Integer.valueOf(1);
```

Comment:
The code is correct as it was; you've added unnecessary boxing,  Explicit manual boxing is unnecessary starting with Java 5 and can safely be removed.  Please see if there are other occurrences.  Thanks.